docs/deprecated.md

```diff
@@ -0,0 +1,37 @@
+# Deprecated functionality
+
+Deprecated features are features that lwm2m-node-lib stills support but that are
+not maintained or evolved any longer. In particular:
+
+-   Bugs or issues related with deprecated features and not affecting
+    any other feature are not addressed (they are closed in github.com
+    as soon as they are spotted).
+-   Documentation on deprecated features is removed from the repository documentation.
+    Documentation is still available in the documentation set associated to older versions
+    (in the repository release branches).
+-   Deprecated functionality is eventually removed from lwm2m-node-lib. Thus you
+    are strongly encouraged to change your implementations using lwm2m-node-lib
+    in order not rely on deprecated functionality.
+
+A list of deprecated features and the version in which they were deprecated follows:
+
+* Support to Node.js v4 in lwm2m-node-lib 1.1.0.
+
+## Using old lwm2m-node-lib versions
+
+Although you are encouraged to use always the newest lwm2m-node-lib version, take into account the following
+information in the case you want to use old versions:
+
+* Code corresponding to old releases is
```

Comment:
Better to use examples meaninfull in the case of this repo. Just replace 0.23.0 by 0.5.0 for instance.

Reply:
Fixed with 1bbaed44caf152c9cd022154525162218accdfab